packages/core/useFavicon/index.ts

```diff
@@ -0,0 +1,29 @@
+import { isString } from '@vueuse/shared'
+import { ref, watch, Ref, ComputedRef, isRef } from 'vue-demi'
+
+export function useFavicon(
+  newIcon: Ref<string> | ComputedRef<string> | string | null = null,
+  options: { baseUrl?: string; rel?: string } = {},
+) {
+  const {
+    baseUrl = '/',
```

Comment:
```suggestion
    baseUrl = '',
```

Maybe it's better to be empty by default. Users can set `icon` to `/favicon.ico` more naturally as they will do in html.